app/src/main/java/ch/epfl/sweng/swenggolf/offer/CreateOfferActivity.java

```diff
@@ -349,6 +393,78 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
     }
 
+
+    public static class DatePickerFragment extends DialogFragment {
```

Comment:
If it is only used here, this class should be private.

Reply:
This class should be public and static.

Reply:
ok